commons/src/main/java/org/infinispan/commons/util/Util.java

```diff
@@ -475,6 +476,13 @@ public static String toStr(Object o) {
          return printArray((byte[]) o, false);
       } else if (o == null) {
          return "null";
+      } else if (o.getClass().isArray()) {
```

Comment:
You're missing some more primitives here, I'm getting a `ClassCastException` in `MemoryBasedEvictionFunctionalTest.testDoubleArray`.